src/postprocess/DOS.jl

```diff
@@ -0,0 +1,41 @@
+# Compute densities of states
+
+# IDOS (integrated)
+# N(ε) = sum_n f_n = sum_n f((εn-ε)/T)
+# DOS
+# D(ε) = N'(ε)
+# LDOS (local)
+# LD = sum_n f_n ψn
+
+using ForwardDiff
+
+"""
+Total density of states at energy ε
+"""
+function DOS(ε, basis, orben; smearing=basis.model.smearing, T=basis.model.temperature)
```

Comment:
Maybe also have a version that takes an `scfres`, because that directly contains ε, basis and orben?

Reply:
I have that in my radar anyway, we should factor a number of objects into a proper `ElectronicState` struct or something, but I'm not 100% sure about the API (ie should psi be with the hamiltonian of which it is the ground state, or with the hamiltonian H[psi] it creates?). Let's discuss that next time.

Reply:
Ja not sure about the name, but I agree that would be a good thing to have. Let's talk next week.